content/en/blog/2024/getting-started-with-otelsql/index.md

```diff
@@ -0,0 +1,428 @@
+---
+title: 'Getting Started with otelsql: OpenTelemetry instrumentation for Go SQL'
+linkTitle: Getting Started with otelsql
+date: 2024-02-24
+author: '[Sam Xie](https://github.com/XSAM) (Cisco)'
+draft: true # TODO: remove this line once your post is ready to be published
+---
+
+[otelsql](https://github.com/XSAM/otelsql) is an instrumentation library for the
+[`database/sql`](https://pkg.go.dev/database/sql) library of Go programming
+language. It generates traces and metrics data while the application interacts
+with the database via `database/sql` library. By that, it allows you to identify
+errors or slowdowns in your SQL queries that potentially impact the performance
+of your application.
+
+This post provides a quick-start guide for using this library.
+
+## Getting Started
+
+otelsql is basically a wrapper layer for interfaces from `database/sql`. When
+users use the wrapped database interfaces, the otelsql generates the telemetry
+data and passes operations to the underlying database.
+
+In the following, you are going to use the docker compose to run an
+otel-collector example from the otelsql repository. This example uses a MySQL
```

Comment:
```suggestion
In the following, you are going to use docker compose to run an
example from the otelsql repository. This example uses a MySQL
```

Reply:
This paragraph specifies the `otel-collector` example that we are going to use. Is there any reason to remove the word `otel-collector`?

Reply:
then you should say "the `otel-collector` example" to make it clearer that this is the name of that example and not just an example that uses the OpenTelemetry Collector